server/conf/routes

```diff
@@ -85,6 +85,11 @@ GET     /admin/programs/:programId/applications/:applicationId            contro
 GET     /admin/programs/:programId/applications/:applicationId/download   controllers.admin.AdminApplicationController.download(request: Request, programId: Long, applicationId: Long)
 GET     /admin/demographics                                               controllers.admin.AdminApplicationController.downloadDemographics()
 
+# Admin API key management
+GET     /admin/apiKeys                              controllers.admin.AdminApiKeysController.index()
+GET     /admin/apiKeys/new                          controllers.admin.AdminApiKeysController.newOne(request: Request)
```

Comment:
question: newOne and create don't read too differently from me, is it normal in REST to not have the phrasing of them be more indicative of what you're getting? or does Get/Post largely make that distinction.  EG a GET wouldn't have side effects so newOne can be inferred to be a page that facilitates making a new one.